config/config.go

```diff
@@ -168,7 +167,36 @@ func ReadConfigFile(configFile string, inputPrefix, outputPrefix string) (*Contr
 		return nil, fmt.Errorf("trouble reading config file %q: %w", absConfigFile, err)
 	}
 
-	return ReadConfig(log, yamlFile, inputPrefix, outputPrefix)
+	config, err := ReadConfig(log, yamlFile, inputPrefix, outputPrefix)
+
+	items, _ := ioutil.ReadDir(util.AbsolutePath(inputPrefix, configDir))
+	for _, item := range items {
+		fileExtension := filepath.Ext(item.Name())
+		if fileExtension == ".yaml" || fileExtension == ".yml" {
```

Comment:
Probably worth to do

```suggestion
		if (fileExtension == ".yaml" || fileExtension == ".yml") && !item.IsDir() {
```